docs/content/en/usage/features.md

```diff
@@ -414,14 +414,15 @@ Similar Findings
 DefectDojo allows you to maintain your security SLAs and automatically
 remind teams whenever a SLA is about to get breached, or is breached.
 
-Simply indicate in the `System Settings` for each severity, how many
-days teams have to remediate a finding.
+To use with SLAs, open the `System Settings` page and click the 'Enable Finding SLAs' checkbox. 
```

Comment:
In order to apply SLAs to findings, open `System Settings` and check 'Enable Finding SLAs'.

Reply:
Thank you!